yapf/yapflib/file_resources.py

```diff
@@ -23,6 +23,27 @@
 from lib2to3.pgen2 import tokenize
 
 from yapf.yapflib import py3compat
+from yapf.yapflib import style
+
+
+LOCAL_STYLE = '.style.yapf'
```

Comment:
Sorry, I thought you were referring to DEFAULT_STYLE, which I did move.

Regarding comments, that is one of things I don't like (or don't understand) about github review system.  I pushed my updated change with push -f .. so the SHA changed, I could have made tweaks in the new change but I don't donn't to pullute history with lots tweeks even before a change lands.  Perhaps you know some other better why to do this than push -f?